Packs/ApiModules/Scripts/MicrosoftApiModule/MicrosoftApiModule.py

```diff
@@ -127,6 +128,14 @@ def http_request(
         if is_response_empty_and_successful and return_empty_response:
             return response
 
+        # Handle 404 errors instead of raising them as exceptions:
+        if response.status_code == 404:
+            try:
+                error_message = response.json()
+            except Exception:
+                error_message = 'Page Not Found - 404 Response'
```

Comment:
```suggestion
                error_message = 'Not Found - 404 Response'
```
in this case it is not a page